automatic update not working

Since a week the automatic update of this repository has stopped.

The tasks timeout after 6 hours.

From a recent failure:

```
2023-12-20T16:14:13.2567213Z Requested labels: ubuntu-latest
...
2023-12-20T17:06:14.4931009Z INFO: cmake-mode: prefetched repository 1eed682d7cca9bb2c2b0709a6c3202a3b08613b2 166i1j8xlv4m82g2dyjl02kdx3yybsyd3jv3dq6f7n94widv3yq6
2023-12-20T22:14:31.5730926Z ##[error]The operation was canceled.
```

Looks like normal movement for close to an hour, then progress stalls with no obvious error before eventually timing out. Feels vaguely like getting hit by rate-limiting or some other form of broken checkout without the error getting surfaced in the log output.

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -70,6 +70,7 @@ jobs:
 
     - name: Update inputs
       run: |
+        export GIT_PAGER=
         ./update ${{ matrix.repo }}
 
     - name: Push commit with updated inputs

